tmva/tmva/inc/TMVA/DNN/Architectures/Cpu.h

```diff
@@ -103,6 +103,44 @@ class TCpu
                                             const TCpuMatrix<Scalar_t> & weights_state, // HxH
                                             const TCpuMatrix<Scalar_t> & input,  // BxD
                                             TCpuMatrix<Scalar_t> & input_gradient);
+	
+  /** Backward pass for LSTM Network */
+    static Matrix_t & LSTMLayerBackward(TCpuMatrix<AFloat> & state_gradients_backward,
```

Comment:
I think it is because we need to be able to call it from inside CellBackward in LSTMLayer.h without instantiating the TCpu class. RecurrentLayerBackward() function (for RNN backward pass) has also been defined as static in the header file.

Reply:
Then `RecurrentLayerBackward()` for RNN backward pass seems to be wrong, too... @lmoneta ?